Automattic/wp-calypso
# Add the basics for a new Donations block.

This PR adds a Donations block.

See: https://github.com/Automattic/wp-calypso/issues/42856

**Testing Instructions**
* Apply this branch to a local install and sync to your sandbox with `yarn dev --sync`.
* Apply D44450-code if it hasn't yet been merged.
* Sandbox the API and a test site.
* Create a new post at `https://:site/wp-admin/post-new.php`.
* Insert the Donations block. Be sure you can use both the inserters (`[+]`) and the `/` shortcut.
* Verify `Donations block` displays in both the the editor and on the front-end.
